openjdk.test.jlm/build.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<project name="openjdk.test.jlm" default="build">
+
+	<!-- Set default for source_root.  -->
+	<property name="source_root" location="../"/>
+
+	<!-- Set default for STF location.  -->
+	<!-- Assumes source repositories are checked out under a common root and the repository
+		has been built in-situ - i.e.
+		stf repository: /home/userid/git/stf
+		openjdk-systemtest repository: /home/userid/git/openjdk-systemtest
+		If the stf repository has been cloned or built to a different location, the stf_root property
+		must be set on the command line - e.g. -Dstf_root=/my_stf_build_dir
+	-->
+	<property name="stf_root" location="${source_root}/../stf"/>
+
+	<!-- Import settings used by multiple projects.  -->
+	<import file="${source_root}/openjdk.build/include/top.xml"/>
+		
+	<!-- For modularity you need a directory above the package structure to hold the module.java file -->
+	<property name="module" value="test.jlm" />
+	<property name="src_dir" value="src/${module}" />
+	<property name="bin_dir" value="bin" />
+
+	<property name="jar_file" value="${bin_dir}/${module}.jar" />
+	
+	<!-- We need junit and stf to compile this project. -->
+	<path id="project.class.path">
+		<path refid="junit.class.path" />
+		<path refid="stf.class.path" />
+	</path>
+	
+	<target name="build" depends="build-no-natives, build-natives">
+	</target>
+
+	<target name="build-no-natives" depends="check-prereqs, build-archives">
+	</target>
+
+	<target name="build-natives">
+	</target>
+
+	<target name="build-archives" depends="build-jar">
+	</target>
+
+	<target name="build-jar" depends="build-java, create-bin-dir">
+		<jar destfile="${jar_file}">
+			<fileset dir="${bin_dir}" includes="**/*.class" />
+		</jar>
+	</target>
+
+	<target name="build-java" depends="check-prereqs, create-bin-dir">
+		 <!--
+		 	The Ant javac task only checks time dependencies between .java files and their .class files,
+			so fails to recompile in situations such as the signatures of a dependent method changing.
+			The depend target checks the dependencies and deletes any .class files older than the files
+			which depend on them, thereby ensuring recompilation.
+		-->
+		<depend srcdir="${src_dir}" destdir="${bin_dir}" classpathref="project.class.path">
+			<include name="**/*.java"/>
+		</depend>
+		<javac srcdir="${src_dir}"
+			   destdir="${bin_dir}"
+			   fork="true"
+			   executable="${java8_compiler}"
```

Comment:
We should not hard code `${java8_compiler}`, what about java9, 10, 11?
I think the code works because in top.xml, it happens to set java8_compiler and java9_compiler to `java.home` if it is not defined. https://github.com/AdoptOpenJDK/openjdk-systemtest/blob/0b730da942494ff5c4aea03593dffd8cf9252ae2/openjdk.build/include/top.xml#L57
For example, the output for jdk10 testing is very confusing:
```
17:55:24      [echo] java8_home set to /home/jenkins/workspace/openjdk10_j9_systemtest_x86-64_linux/openjdkbinary/j2sdk-image/bin/..
17:55:24      [echo] java8_compiler set to /home/jenkins/workspace/openjdk10_j9_systemtest_x86-64_linux/openjdkbinary/j2sdk-image/bin/javac
```
https://ci.adoptopenjdk.net/view/System%20tests/job/openjdk10_j9_systemtest_x86-64_linux/6/consoleFull

This is may not be the scope of this PR, but top.xml should be fixed. There is no need to have separate variables for java compiler.

Reply:
Thanks! Will create separate PR for this.